server/test/e2e/setup-docker.ts

```diff
@@ -0,0 +1,26 @@
+import { PostgreSqlContainer } from '@testcontainers/postgresql';
+
+export default async () => {
+  if (process.env.DB_HOSTNAME === undefined) {
+    const pg = await new PostgreSqlContainer('postgres')
+      .withExposedPorts(5432)
+      .withDatabase('immich')
+      .withUsername('postgres')
+      .withPassword('postgres')
+      .withReuse()
+      .start();
+
+    process.env.DB_URL = pg.getConnectionUri();
+  }
+  process.env.NODE_ENV = 'development';
+  process.env.TYPESENSE_ENABLED = 'false';
+  process.env.IMMICH_MACHINE_LEARNING_ENABLED = 'false';
+  process.env.IMMICH_TEST_ENV = 'true';
+  process.env.DB_HOSTNAME = 'immich-database-test';
+  process.env.DB_USERNAME = 'postgres';
+  process.env.DB_PASSWORD = 'postgres';
+  process.env.DB_DATABASE_NAME = 'e2e_test';
+  process.env.UPLOAD_LOCATION = './upload';
+  process.env.VITE_SERVER_ENDPOINT = 'http://localhost:2283/api';
+  process.env.TZ = 'Z';
```

Comment:
If we're using docker we're setting these in the compose file right? Why do we need them in both places? If we want to use them in two places, maybe we can add back `test.env` and then load it here via `dotenv.config({ envFile: 'test.env' })`

Reply:
We're not always using docker. You can also run this from npm run test:e2e, so we need to set it here